### ISSUE TITLE
关于模型有几个问题想请教一下

### ISSUE THREAD

**Issue by lzl19971215** [CQUPT-CaiKe/PBERT-MTL#2]
您好，看了您文章里的方法，结合着看了一下代码，有以下几个问题想请教一下

1. 您修改了loss，参考的是logit adjustment（https://arxiv.org/abs/2007.07314）
这里结合训练的代码，好像只有在eval_flag为true，也就是测试的时候使用了logit adjust，这个结合原文，应该是用的Post-hoc logit adjustment吧？但是为什么只计算了loss，没有用调整的logit来进行预测呢？
![image](https://github.com/CQUPT-CaiKe/PBERT-MTL/assets/63151530/cfdb4223-ba7b-4518-924c-58f339bc81c6)

2. 如果使用的是logit adjusted softmax cross-entropy来直接训练，那么loss里的代码是否有问题？原文是+tau * logP，这里是- tau * logP，而且好像先验概率也没有归一化。
![image](https://github.com/CQUPT-CaiKe/PBERT-MTL/assets/63151530/c537f3bb-1f25-4519-b61e-bc6a38cf47ec)
![image](https://github.com/CQUPT-CaiKe/PBERT-MTL/assets/63151530/a239cf42-730c-4113-9d79-6a049b5e8bc1)

还望解答，感谢！
